sysutils/xdg-utils/Portfile

```
@@ -1,9 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           gitlab 1.0
 
+gitlab.instance     https://gitlab.freedesktop.org
+gitlab.setup        xdg xdg-utils v1.2.1
```

Don't combine the tag prefix with the version; this is the reason why livecheck is failing.
```suggestion
gitlab.setup        xdg xdg-utils 1.2.1 v
```